dlc/Cargo.toml

```diff
@@ -9,7 +9,7 @@ version = "0.4.0"
 
 [dependencies]
 bitcoin = {version = "0.29.2"}
-miniscript = "8.0.0"
+miniscript = { version = "9.0", features = ["serde", "std"], default-features = false }
```

Comment:
What is the reason for this change? Also it'd be better to put the serde feature behind the `use-serde` feature flag.